src/components/calcite-sortable-list/calcite-sortable-list.tsx

```diff
@@ -0,0 +1,169 @@
+import Sortable from "sortablejs";
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Prop,
+  State,
+  h
+} from "@stencil/core";
+
+/**
+ * @slot - A slot for adding sortable items
+ */
+@Component({
+  tag: "calcite-sortable-list",
+  styleUrl: "./calcite-sortable-list.scss",
+  shadow: true
+})
+export class CalciteSortableList {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * The class on the handle elements.
+   */
+  @Prop() handleSelector = ".handle";
```

Comment:
So is the `.handle` going into the shadowDom and the drag and drop supports it? Just wondering why it isn't just `calcite-handle`.

Reply:
the host can't receive focus, its basically a div. The button is the focus item and the cornerstone of the handle functionality. When using the keyboard, when sortable sorts the items, the parent of the handle will be removed from the dom and re-inserted. This causes it to lose focus so we have to re-apply focus. That focus needs to go on the button, not the host.

Reply:
@pr3tori4n does Sortable find the `.handle` in the shadowDOM? Is that how this works?

Reply:
fixme: Lets change this to `calcite-handle` by default with the `setFocus()` method.